documents/readthedoc/docs/source/index.rst

```diff
@@ -56,6 +56,7 @@ Table of Contents
    Solutions/tensorflow-serving-cluster/index.rst
    Solutions/horizontal-federated-learning/hfl.md
    Solutions/grpc-ra-tls/index.md
+   Solutions/ehsm-kms/ehsm.md
```

Comment:
do you mind changing the main page name from ehsm.md to index.md?

Reply:
ok